docker.go

```diff
@@ -85,10 +112,14 @@ func (d *DockerManager) getService(id string) (*Service, error) {
 func (d *DockerManager) eventCallback(event *dockerclient.Event, ec chan error, args ...interface{}) {
 	//log.Printf("Received event: %#v %#v\n", *event, args)
 
+	s, s_err := d.list.GetService(event.Id)
+
 	switch event.Status {
 	case "die", "stop", "kill":
 		// Errors can be ignored here because there can be no-op events.
-		d.list.RemoveService(event.Id)
+		if s_err == nil && !s.Manual {
+			d.list.RemoveService(event.Id)
```

Comment:
One way is to not remove these but mark them as deleted. So if they are restarted they get the manual name again. But unless I get some critical feedback for #31 I'm going to remove the mixing of manual and automatic anyway so this would be much simpler.